CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added support for code:load_abs/1, code:load_binary/3
 - Added support for loading / closing AVMPacks at runtime
 - Added support for ESP-IDF v5.x
+- Added support for `atomvm:set_time_of_day/2`
```

Comment:
We may want to rename this `clock_settime` or even `posix_clock_settime`.

Reply:
`atomvm:posix_clock_settime/2`